### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by StarMindz**
I created this branch to add a contact form to the mobile portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -147,8 +147,31 @@ <h2 id="skill">Skills</h2>
...
+
+        <section id="Contact-us">
+            <h4 class="interest">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+            <div class="form1">
+            <form action="https://formspree.io/f/mnqwkpaw" method="POST">
+              <input class="text1" type="text" name="name88" id="name89" placeholder="Full name" required>
+              <br>
+              <input class="text1" type="email" name="email" id="email" placeholder="Email address" required>
+              <br>
+              <input class="text2" type="text" name="text" id="text" placeholder="Enter text here" maxlength="500">
```
Comment (algerina) [inline index.html:159]: - [ ] Kindly add a `maxlength="30"`  to the name input
- [ ] Please, message input should be `<textarea>`, it should be required too.

File: style.css [hunk 2]
```diff
@@ -423,3 +426,65 @@ button {
...
+
+.text1 {
+  border: none;
+  border-bottom: 1px solid black;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin-left: 40px;
```
Comment (algerina) [inline style.css:460]: - [ ] You did a good job however, it is required to use flexbox to layout the form elements

**Review by algerina — CHANGES_REQUESTED** [review]
Hi  @StarMindz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters are passing
- Your project seems good
- You have a descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (StarMindz) [reply style.css]: Thank you @algerina. I've made all changes as requested

**Review by mirouhml — CHANGES_REQUESTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that didn't pass linters checks, please make sure it does before submitting again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by StarMindz** [comment]
@mirouhml I have made the needed changes. Thank you for your review

**Review by muneebulrehman — APPROVED** [review]
Hi @StarMindz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**